backend/pom.xml

```diff
@@ -20,31 +20,71 @@
   <packaging>pom</packaging>
 
   <properties>
-    <junit.version>5.6.0-RC1</junit.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <junit.version>5.7.0-RC1</junit.version>
     <slf4j.version>1.7.30</slf4j.version>
+    <jackson.version>2.11.2</jackson.version>
+    <mockito.version>3.5.10</mockito.version>
+    <assertj.version>3.17.2</assertj.version>
+    <logback.version>1.2.3</logback.version>
+    <commons-compress.version>1.20</commons-compress.version>
+
+    <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
+    <maven-shade-plugin.version>3.2.4</maven-shade-plugin.version>
+    <maven-surefire-plugin.version>3.0.0-M4</maven-surefire-plugin.version>
+    <jacoco-maven-plugin.version>0.8.5</jacoco-maven-plugin.version>
+    <maven-site-plugin.version>3.9.0</maven-site-plugin.version>
+    <versions-maven-plugin.version>2.8.1</versions-maven-plugin.version>
+    <maven-project-info-reports-plugin.version>3.1.1</maven-project-info-reports-plugin.version>
   </properties>
 
 
   <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <version>3.0.0-M3</version>
```

Comment:
If you made variables for all other versions, why not for this one?

Reply:
Human error. Let me rebase the PR